diff --git a/node_modules/youtube-dl-exec/scripts/preinstall.mjs b/node_modules/youtube-dl-exec/scripts/preinstall.mjs
index 738714e..e839e4e 100644
--- a/node_modules/youtube-dl-exec/scripts/preinstall.mjs
+++ b/node_modules/youtube-dl-exec/scripts/preinstall.mjs
@@ -14,9 +14,5 @@ function throwError(message) {
 }
 
 if (process.env.YOUTUBE_DL_SKIP_PYTHON_CHECK) {
-	process.exit(0)
+	// process.exit(0)
 }
-
-if (!(await which('python', { nothrow: true }))) {
-	throwError(new Error('youtube-dl-exec needs Python. Couldn\'t find the `python` binary. Make sure it\'s installed and in your $PATH.. You can skip this check passing `YOUTUBE_DL_SKIP_PYTHON_CHECK=1`'))
-}
